tools/number-converter/src/NumberConverter.vue

```diff
@@ -71,8 +81,16 @@ export default {
 			errorMessage: null,
 			inputNumber: null,
 			result: null,
-			base: 0,
-			baseOptions: [
+			fromBase: 0,
+			toBase: 0,
+			fromBaseOptions: [
```

Comment:
this seems like duplication, can we not store both `to` and `from` in a single `baseOptions` object?

Reply:
yes!

Reply:
let's do that 😄

Reply:
done!